test/unit/runner.spec.js

```diff
@@ -487,6 +487,32 @@ describe('Runner', function() {
         });
```

Comment:
On L450 and L471, can we change the skip condition to be more apparent?
```js
 if (path.sep !== '/') {
```
would become
```js
 if (process.platform === 'win32') {
```

Reply:
yes, but it doesn't need to happen in this PR.